### ISSUE TITLE
[0.52] when applying ESDF input preset, the medkit is said to be not bound

### ISSUE THREAD

**Issue by illwieckz** [UnvanquishedAssets/unvanquished_src.dpkdir#43]
[0.52] when applying ESDF input preset, the medkit is said to be not bound

**Comment by slipher** [comment]
```
teambind humans     hw:a   itemact medkit
teambind humans     hw:1   itemtoggle blaster
teambind humans     hw:r   reload
teambind humans     hw:a   +sprint
```

**Comment by illwieckz** [comment]
Ah right, `hw:a` is bound two time! I'll fix that.

**Comment by illwieckz** [comment]
fixed by e1088d0

it looks like I was confused by my azerty keyboard 🤦‍♀️
